### ISSUE TITLE
Get-HPOVOSDeploymentServer  Error: Cannot convert value 

### ISSUE THREAD

**Issue by sinelshchikov** [HewlettPackard/POSH-HPOneView#259]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
Get-HPOVOSDeploymentServer should return the list of all deployment servers

### Actual Behavior
```
PS C:\Users\Administrator> Get-HPOVOSDeploymentServer
Cannot convert value "@{type=DeploymentManager; mgmtNetworkUri=/rest/ethernet-networks/747fe69e-0623-40fe-8bcf-67f0ed26dc7b;
primaryIP=fe80:0:0:0:b365:de64:a174:438;
primaryActiveAppliance=/rest/deployment-servers/image-streamer-appliances/56577a5e-d599-41eb-872c-f7e17f0caea5;
eTag=2017-03-22T21:00:41.778Z; created=2017-03-22T21:00:41.778Z; modified=2017-03-22T21:00:41.778Z;
primaryUUID=fd09303d-53f2-4696-917e-3bdefdd46f48; checksum=F8E31F5331502BA55222942193E2FC54A3A689093DE5E77AB8DD8C70FF14C868;
deplManagersType=Image Streamer; primaryUIUri=/rest/deployment-appliances/d7f5cc63-9c26-4027-896d-300797caa751;
primaryIPV4=10.1.0.76; primaryClusterStatus=Pending; primaryClusterName=Image Streamer-appliance-1-primary; description=;
category=deployment-managers; status=OK; uri=/rest/deployment-servers/ed63c740-d0ea-454a-9cef-b9cc5e4a7a80; name=Image Streamer;
id=ed63c740-d0ea-454a-9cef-b9cc5e4a7a80; state=Connected; ApplianceConnection=}" to type "HPOneView.Appliance.OSDeploymentServer".
Error: "Cannot convert the "@{type=DeploymentManager; mgmtNetworkUri=/rest/ethernet-networks/747fe69e-0623-40fe-8bcf-67f0ed26dc7b;
primaryIP=fe80:0:0:0:b365:de64:a174:438;
primaryActiveAppliance=/rest/deployment-servers/image-streamer-appliances/56577a5e-d599-41eb-872c-f7e17f0caea5;
eTag=2017-03-22T21:00:41.778Z; created=2017-03-22T21:00:41.778Z; modified=2017-03-22T21:00:41.778Z;
primaryUUID=fd09303d-53f2-4696-917e-3bdefdd46f48; checksum=F8E31F5331502BA55222942193E2FC54A3A689093DE5E77AB8DD8C70FF14C868;
deplManagersType=Image Streamer; primaryUIUri=/rest/deployment-appliances/d7f5cc63-9c26-4027-896d-300797caa751;
primaryIPV4=10.1.0.76; primaryClusterStatus=Pending; primaryClusterName=Image Streamer-appliance-1-primary; description=;
category=deployment-managers; status=OK; uri=/rest/deployment-servers/ed63c740-d0ea-454a-9cef-b9cc5e4a7a80; name=Image Streamer;
id=ed63c740-d0ea-454a-9cef-b9cc5e4a7a80; state=Connected; ApplianceConnection=}" value of type
"System.Management.Automation.PSCustomObject" to type "HPOneView.Appliance.OSDeploymentServer"."
At C:\Program Files\WindowsPowerShell\Modules\HPOneView.300\HPOneView.300.psm1:62785 char:6
+                     [void]$_ImageStreamerCollection.Add([HPOneView.Appliance.OSDeploymentServer ...
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvalidCastConstructorException

PS C:\Users\Administrator>
```

### Steps to reproduce

Add a Deployment Server on a synergy frame
Verify that the DS is added and state is "connected"
Open "HPE OneView 3.00 PowerShell Library" and run "Get-HPOVOSDeploymentServer"

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`): 3.0.1264.2772
HPE OneView Appliance Version:  3.00.07-0286834
Output from `$PSVersionTable` on your Windows Host:
Name                           Value
----                           -----
PSVersion                      4.0
WSManStackVersion              3.0
SerializationVersion           1.1.0.1
CLRVersion                     4.0.30319.42000
BuildVersion                   6.3.9600.17400
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0}
PSRemotingProtocolVersion      2.2

**Comment by ChrisLynchHPE** [comment]
This will be fixed in the next release of the library.  Those using PowerShell 5.0 or newer are not affected by this defect.

**Comment by sinelshchikov** [comment]
Chris, could you please add a fix for "Get-HPOVImageStreamerAppliance" as well ?

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 3.0.1293.3770 ](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1293.3770).

Closing.
